src/basic/log.c

```diff
@@ -49,6 +49,7 @@ static int kmsg_fd = -1;
 static int journal_fd = -1;
 
 static bool syslog_is_stream = false;
+bool error_check = false; /* to error out when unit verification fails */
```

Comment:
This needs to have a proper meaningful name that is appropriate for a global variable. It also needs to be declared in `log.h`.